src/test/java/com/wikia/webdriver/common/core/configuration/Configuration.java

```diff
@@ -107,6 +107,8 @@ public static Credentials getCredentials() {
   public static String getEnvType() {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) The Cyclomatic Complexity of this method "getEnvType" is 11 which is greater than 10 authorized. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AMethodCyclomaticComplexity)